routers/api/v1/repo/repo.go

```diff
@@ -88,6 +118,22 @@ func Search(ctx *context.APIContext) {
 		return
 	}
 
+	var sortMode = ctx.Query("sort")
+	if len(sortMode) > 0 {
+		var sortOrder = ctx.QueryInt("order") // will default to 0, i.e. ascending
```

Comment:
It would be better if order would be string (`asc`/`desc`)

Reply:
Fixed. asc/desc does make more sense than 0/1